test-commons/src/main/java/org/sonarsource/analyzer/commons/checks/verifier/internal/InternalIssueVerifier.java

```diff
@@ -135,6 +135,12 @@ public void assertOneOrMoreIssues() {
     assertIssues(true);
   }
 
+  @Override
+  public void assertNoIssuesRaised() {
+    Report report = buildReport();
+    assertEquals("ERROR: Found " + report.getActualIssueCount() + " unexpected issues.", 0, report.getActualIssueCount());
```

Comment:
Can we add an explicit message about the fact that we expect no issues at all? My fear would be that someone uses the method, then there are issues raised, and they look into the sample file, assuming we called `verifyIssues()`, having a hard time figuring out that it is the expectation that is different to theirs.

Reply:
What about "ERROR: No issues were expected, but some were found."?